08-compiler-features.tex

```diff
@@ -228,7 +228,7 @@ \section{Completion}
 haxe --connect 6000 myproject.hxml
 \end{lstlisting}
 
-Please note that you can use \ic{--cwd} at the first sent commandline parameter to change the Haxe server's current working directory. Usually class paths and other files are relative to your project.
+Please, note that you can use the parameter \ic{--cwd} at the first sent command line to change the Haxe server's current working directory. Usually class paths and other files are relative to your project.
```

Comment:
You are right again, thank you. I guess it is a portuguese speaker bias. Please is translated as 'by favour', so the comma would make some sense there.